views/item/new.ejs

```diff
@@ -1,35 +1,99 @@
 <%- contentFor('title') %> EcoFoodie
 <%- contentFor('head') %>
+
+<link rel="stylesheet" type="text/css" href="/stylesheets/itemNew.css">
 <script type="application/javascript" src="/scripts/uploader.js"></script>
 <%- contentFor('body') %>
-<h1>New Items</h1>
+
+
+
+<img src="/images/greenArrow.png" alt="green arrow" style="width:128px;height:128px;">
+<img src="/images/EcoFoodieLOGO.png" alt="Eco Foodie Logo" style="width:128px;height:128px;">
+
 
 <form method="post" action="/item">
   <div id="other" class="form-group" style="display: none;">
     <input type="text" class="form-control" name="other" value="" placeholder="Please describe...">
   </div>
 
-  <div class="form-group">
+<!--  <div class="form-group">
     <label>Restaurant</label>
     <textarea class="form-control" rows="2" name="restaurantId" ></textarea>
-  </div>
+  </div> -->
 
   <div class="form-group">
-    <label>Picture</label>
+    <h3>Picture</h3>
     <input type="file" class="form-control" data-name="photo" />
   </div>
 
-  <div class="form-group">
-    <label>Bin</label>
-    <input type="textarea" class="form-control" name="bin" value="" />
+
+  <h2>What is this photo about?</h2>
+  <br>
+
+  <select id="category">
+  <option value="Plastic">Plastic</option>
+  <option value="LandFill">LandFill</option>
+  <option value="Recycle">Recycle</option>
+  </select>
```

Comment:
The Category options should not be written directly into this file.

Rather, we should _load_ the Categories from the database. Can you go back to your Categories routes and views to figure out how to load the categories in the items route for this page, and then display the categories in this dropdown list?